ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/PaywallDataExtensions.kt

```diff
@@ -0,0 +1,102 @@
+package com.revenuecat.purchases.ui.revenuecatui.extensions
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.platform.LocalContext
+import androidx.core.os.LocaleListCompat
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.paywalls.PaywallColor
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.data.processed.PaywallTemplate
+import java.net.URL
+
+/***
+ * Default [PaywallData] to display when attempting to present a [PaywallView] with an offering that has no paywall
+ * configuration, or when that configuration is invalid.
+ */
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.Companion.createDefault(packages: List<Package>): PaywallData {
+    return PaywallData.createDefaultForIdentifiers(packages.map { it.identifier })
+}
+
+@Composable
+@ReadOnlyComposable
+private fun PaywallData.Companion.createDefaultForIdentifiers(packageIdentifiers: List<String>): PaywallData {
+    val context = LocalContext.current
```

Comment:
Note that the fact that we have this in the same place that we calculate the `PaywallData` means that, whenever there is a configuration change, this and anything depending on this function will get recompositioned.

Additionally, I'm not totally sure how we're planning to use this, since the paywall data is currently obtained/stored by the view model, which can't access composable functions.

Reply:
Made the change in 6ce47582792b7aec883283d4ab2471428306d8f9 and these are not composable anymore